providers/google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/binders/MultipartUploadBinder.java

```diff
@@ -57,6 +57,7 @@
       Part jsonPart = Part.create("Metadata", jsonPayload, new Part.PartOptions().contentType(APPLICATION_JSON));
       Part mediaPart = Part.create(template.name(), payload, new Part.PartOptions().contentType(contentType));
 
+      request.resetPayload(/*release=*/ false);
```

Comment:
This is not ideal but I don't understand what the `RestAnnotationProcessor` is doing.

Reply:
Sorry I'm a bit late here. Do you have a set of specific lines/methods on the processor? Perhaps I can help fix those instead.